packages/models/index.ts

```diff
@@ -1,7 +1,8 @@
-export type { Expectation } from './expectation';
-export type { SkillLevel } from './level';
-export type { NewApplicant } from './newApplicant';
-export type { Opportunity } from './opportunity';
-export type { HeroImage, Project } from './project';
-export type { Skill } from './skill';
-export type { User } from './user';
+export { Expectation } from "./expectation";
+export { SkillLevel } from "./level";
+export { NewApplicant } from "./newApplicant";
+export { Opportunity } from "./opportunity";
+export { HeroImage, Project } from "./project";
+export { Skill } from "./skill";
+export { User } from "./user";
+export { Idea } from './idea';
```

Comment:
why did we change this from `export type`  to just `export` ?

Reply:
Thanks. Not sure this was intentional.